src/views/account-details/account-details.view.tsx

```diff
@@ -1,30 +1,96 @@
-import { getFixedTokenAmount, getTokenAmountInPreferredCurrency } from "../../utils/currencies";
-import Container from "../shared/container/container.view";
-import { changeHeader } from "../../store/global/global.actions";
-import TransactionActions from "../shared/transaction-actions/transaction-actions.view";
-import ExitCardList from "../shared/exit-card-list/exit-card-list.view";
-import FiatAmount from "../shared/fiat-amount/fiat-amount.view";
-import TokenBalance from "../shared/token-balance/token-balance.view";
-import InfiniteScroll from "../shared/infinite-scroll/infinite-scroll.view";
-import { resetState } from "../../store/account-details/account-details.actions";
-import { AUTO_REFRESH_RATE } from "../../constants";
-import { getAccountBalance } from "../../utils/accounts";
-import { mergeExits } from "../../utils/transactions";
-import * as storage from "../../utils/storage";
+import useAccountDetailsStyles from "src/views/account-details/account-details.styles";
+import TransactionList from "src/views/account-details/components/transaction-list/transaction-list.view";
+import Container from "src/views/shared/container/container.view";
+import Spinner from "src/views/shared/spinner/spinner.view";
+import TransactionActions from "src/views/shared/transaction-actions/transaction-actions.view";
+import ExitCardList from "src/views/shared/exit-card-list/exit-card-list.view";
+import FiatAmount from "src/views/shared/fiat-amount/fiat-amount.view";
+import TokenBalance from "src/views/shared/token-balance/token-balance.view";
+import InfiniteScroll from "src/views/shared/infinite-scroll/infinite-scroll.view";
+import * as globalThunks from "src/store/global/global.thunks";
+import { GlobalState } from "src/store/global/global.reducer";
+import { changeHeader } from "src/store/global/global.actions";
+import * as accountDetailsThunks from "src/store/account-details/account-details.thunks";
+import { AccountDetailsState } from "src/store/account-details/account-details.reducer";
+import { resetState } from "src/store/account-details/account-details.actions";
+import { getFixedTokenAmount, getTokenAmountInPreferredCurrency } from "src/utils/currencies";
+import { getAccountBalance } from "src/utils/accounts";
+import { mergeExits } from "src/utils/transactions";
+import * as storage from "src/utils/storage";
+import { AsyncTask, isAsyncTaskDataAvailable } from "src/utils/types";
+import { AUTO_REFRESH_RATE } from "src/constants";
+import { Theme } from "src/styles/theme";
+//domain
+import {
+  FiatExchangeRates,
+  HermezAccount,
+  HermezWallet,
+  HistoryTransaction,
+  isPendingDeposit,
+  PendingDeposit,
+  PoolTransaction,
+  TimerWithdraw,
+  Token,
+} from "src/domain/hermez";
+import { EthereumNetwork } from "src/domain/ethereum";
+import { AppDispatch, AppState } from "src/store";
+import { Exits } from "src/persistence";
+
+type AccountDetailsStateProps = AccountDetailsState &
```

Comment:
I don't think it's a good idea to intersect here with `AccountDetailsState`. The shape of `AccountDetailsState` could differ from `AccountDetailsStateProps` in the future.

Reply:
The idea is that they should be the same because we create a `store` for these views and we are bring that data directly from there

Reply:
@elias-garcia let us know your opinion in this case and HomeState

Reply:
> The idea is that they should be the same because we create a `store` for these views

Yeah, it's true that we have a slice of the store for each view but still, it somehow feels weird to me to use that slice of the store to intersect with other props of the view. I recognize though that it's a debatable detail, probably biased by my personal taste. Let's see what @elias-garcia thinks.

The question that arises now from my OCD alert system is: If we accept this pattern, should we apply it to all views? Currently, it's only present `AccountDetails` and `Home`.